### ISSUE TITLE
OBJECT Directives not being mapped to Java annotations

### ISSUE THREAD

**Issue by TGNThump** [kobylynskyi/graphql-java-codegen#660]
## Issue Description

OBJECT Directives are not being mapped to Java annotations

## Steps to Reproduce

```java
directive @Document(
    collection: String
) on OBJECT

type Test @Document(collection: "test") {
  name: String
}
```

## Expected Result

```java
@javax.annotation.processing.Generated(
    value = "com.kobylynskyi.graphql.codegen.GraphQLCodegen",
    date = "2021-04-19T15:35:57+0100"
)
@org.springframework.data.mongodb.core.mapping.Document("test")
public class Test implements java.io.Serializable {
...
```

## Actual Result

```java
@javax.annotation.processing.Generated(
    value = "com.kobylynskyi.graphql.codegen.GraphQLCodegen",
    date = "2021-04-19T15:35:57+0100"
)
public class Test implements java.io.Serializable {
...
```

## Your Environment and Setup

* **graphql-java-codegen version**: *5.0.0*
* **Build tool**: *Maven*
* **Mapping Config**:

```xml
...
 <directiveAnnotationsMapping>
    <Document>
        <annotation>org.springframework.data.mongodb.core.mapping.Document({{collection}})</annotation>
    </Document>
</directiveAnnotationsMapping>
...
```

**Comment by robbertnoordzij** [comment]
I am able to reproduce it. In the annotation processing for a type, the directive is not added/parsed.

For now (I understand it is not ideal) you can add an customAnnotation to for each type:

```xml
                            <customAnnotationsMapping>
                                <Test>
                                    <annotation>
                                        @org.springframework.data.mongodb.core.mapping.Document("test")
                                    </annotation>
                                </Test>
                            </customAnnotationsMapping>
```

**Comment by robbertnoordzij** [comment]
I think I was able to pin point the problem. However, I am not sure if this is done by accident or on purpose.

**Comment by kobylynskyi** [comment]
Thanks @robbertnoordzij for the changes. Will be released as part of 5.1.0
